libtenzir/builtins/functions/config.cpp

```diff
@@ -19,7 +19,7 @@ namespace {
 class plugin final : public virtual function_plugin {
 public:
   auto name() const -> std::string override {
-    return "config";
+    return "config()";
```

Comment:
This is not valid.

Reply:
Yeah, for some reason I had the operator config.cpp still in my working tree and followed the pattern from `reverse()`. That commit is dropped now.